precompiles/bank/integration_test.go

```diff
@@ -6,11 +6,11 @@ import (
 
 	"github.com/evmos/evmos/v16/precompiles/bank/testdata"
 
-	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
 	"github.com/ethereum/go-ethereum/common"
 	"github.com/evmos/evmos/v16/testutil/integration/evmos/factory"
 	"github.com/evmos/evmos/v16/testutil/integration/evmos/grpc"
 	"github.com/evmos/evmos/v16/testutil/integration/evmos/network"
+	integrationUtils "github.com/evmos/evmos/v16/testutil/integration/evmos/utils"
```

Comment:
NIt: use all lower case to comply with current code style
```suggestion
	integrationutils "github.com/evmos/evmos/v16/testutil/integration/evmos/utils"
```